lychee-lib/src/remap.rs

```diff
@@ -1,20 +1,39 @@
+//! Remapping rules which allow to map URLs matching a pattern to a different
+//! URL.
+//!
+//! # Notes
+//! Use in moderation as there are no sanity or performance guarantees.
+//!
+//! - There is no constraints on remapping rules upon instantiation or during
```

Comment:
```suggestion
//! - There are no constraints on remapping rules upon instantiation or during
```

Reply:
it should be singular case  "is no constraint"